Makefile

```diff
@@ -187,47 +321,88 @@ multi_boot/unk_8E1FE28/unk_8E1FE28.gba: unk_8E1FE28
 multi_boot/unk_8E8490C/unk_8E8490C.gba: unk_8E8490C
 	@$(MAKE) -C multi_boot/unk_8E8490C COMPARE=$(COMPARE)
```

Comment:
I can't test modern at all but at least these targets are not using `MODERN` variable